scenes/menu/preferences.tscn

```diff
@@ -0,0 +1,9 @@
+[gd_scene format=3 uid="uid://byaihtvkgwpt7"]
+
+[node name="Preferences" type="Control"]
```

Comment:
Is this something you added but then threw away? Delete it.

Reply:
weirdly enough i tried to delete it several times and godot kept randomly recreating it. i'll kill it again